examples/vl3_basic/vl3-nse/cmd/vl3_connect.go

```diff
@@ -212,12 +215,36 @@ func (vxc *vL3ConnectComposite) Request(ctx context.Context,
 			}
 		}
 	}
+
+	err := ValidateLabels(requestConnection.Labels)
+	if err != nil {
+		logger.Error(err)
```

Comment:
A lot of if-else-if-else... Any particular reason why we are not returning "return nil, err" here and below?

Reply:
It's the same approach as before, the error is not returned, it's only logged (see the above implementation of the same method)

Reply:
right..we don't want to fail the connection just because of issues with service labels.  Returning error from this means the dataplane doesn't get set up for the client which is the top priority.